packages/react/src/textarea/TextArea.tsx

```diff
@@ -49,7 +49,6 @@ function TextArea(props: TextAreaProps) {
     description,
     errorMessage,
     label,
-    isRequired,
```

Comment:
urelatert til denne PRen, men kopierte denne komponenten som utgangspunkt for Select, og så at vi unødvendig dro ut `isRequired` bare for å sende det videre igjen.